test/test-visually/demo.md

```
@@ -0,0 +1,428 @@
+
+Use backslash \\ to escape special characters.
+
+---
+
+## Conclusion
+
+The **Gruvbox Crisp** theme provides excellent highlighting for all Markdown elements, making your documents both *beautiful* and ***highly readable***.
+
+For more information, visit the [Gruvbox Crisp repository](https://github.com/VatsalSy/zed-gruvbox_custom_themes).
+
+Happy writing! :pencil:
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)